### ISSUE TITLE
Accuracy and Loss

### ISSUE THREAD

**Issue by shan18** [ShankHarinath/DeepSpeech2-Keras#4]
How much accuracy and loss did you get on your validation set with this model?

**Comment by vaibhavsatpathy** [comment]
were you able to train @shan18 ?
If yes then if i may ask what were your inputs and their format?

**Comment by shan18** [comment]
@vaibhavsatpathy Nope, I didn't even try it out, the code is incomplete. You can refer to KerasDeepSpeech repository if you want.
